includes/api/class-wc-rest-cart-controller.php

```diff
@@ -438,6 +438,10 @@ public function update_item( $data = array() ) {
 		$cart_item_key = ! isset( $data['cart_item_key'] ) ? '0' : wc_clean( $data['cart_item_key'] );
 		$quantity      = ! isset( $data['quantity'] ) ? 1 : absint( $data['quantity'] );
```

Comment:
@SHoogland What happens if you change the default value for $quantity to `0`? Is the response different? Is it better or worse?

Reply:
I think it is the same. At least in my case with the product bundle plugin. The problem I am trying to fix/or workaround is that sending a quantity 0 to the "has_enough_stock" function when its a bundle throws a fatal error.

But when I simply remove the item (which is from my point of view the same as setting quantity to 0) there is no error.

Reply:
OK. Just to clarify, does it return the response you need once the item has been removed?

Reply:
Yes, it returns the wp_rest_response from the remove function.